### ISSUE TITLE
Error page roundup

### ISSUE THREAD

**Issue by clash99** [Cadasta/cadasta-platform#629]
Design and content consideration for all 400s, 500s, other? error pages that may display on the Cadasta platform

**Comment by wonderchook** [comment]
Should we get @aliensunrise involved in this from a branding perspective?

**Comment by clash99** [comment]
Plus perhaps a wordsmith too for voice and tone...  I will start by putting together first pass and then we can put them here for review and adjustments.

**Comment by seav** [comment]
Do we want to be cute like GitHub? https://github.com/404

**Comment by wonderchook** [comment]
Probably not "that cute", but I think we can be a little cute.

**Comment by amplifi** [comment]
I'd like to add to this request a custom page to display while the platform is unavailable during scheduled maintenance windows. So far all our processes have been non-disruptive, but that's not always necessarily going to be possible.

**Comment by wonderchook** [comment]
@amplifi makes sense. I think for these sorts of things we should get @aliensunrise involved in the design.

@dpalomino this should also be prioritized in the backlog

**Comment by dpalomino** [comment]
Added in backlog (id 42.00), with low priority.

**Comment by wonderchook** [comment]
@dpalomino to discuss with @aliensunrise

**Comment by SteadyCadence** [comment]
I like this page from product hunt-- 
<img width="738" alt="screen shot 2017-05-16 at 11 21 27 am" src="https://cloud.githubusercontent.com/assets/25337622/26119291/faf069fe-3a29-11e7-94ce-2b56adf19d5f.png">

**Comment by dpalomino** [comment]
I like that @SteadyCadence :) However I think our users' profile are typically less technical so maybe a more "classic" approach like "This is temporarily unavailable, if you continue experiencing this please get in touch with us". 

@aliensunrise, maybe we can catch up re this next or later if you have time? Do you have some ideas?
